agent/build.gradle

```diff
@@ -28,6 +28,13 @@ dependencies {
     }
 
     compile "net.sf.saxon:Saxon-HE:$saxonHEVersion"
+
+    //IKEA Tradfri agent dependencies
+    compile "org.eclipse.californium:californium-core:2.1.0"
+    compile "org.eclipse.californium:scandium:2.1.0"
+    compile "org.apache.commons:commons-lang3:3.9"
+    compile "com.fasterxml.jackson.core:jackson-databind:2.10.2"
```

Comment:
Are we sure we need to explicitly declare these as they are dependencies from elsewhere in the codebase